tests/unit-tests/rest-api/test-class-sensei-rest-api-course-students-controller.php

```diff
@@ -161,7 +161,7 @@ public function testAddUsersToCourses_CourseNotFoundGiven_ReturnsForbiddenRespon
 		$response = $this->server->dispatch( $request );
 
 		/* Assert. */
-		$this->assertSame( 403, $response->get_status() );
+		$this->assertSame( 404, $response->get_status() );
```

Comment:
I'd like to see also check of the new error code (`sensei_course_student_batch_action_missing_course`) in tests for 404.
Maybe something like that:
```
	public function testAddUsersToCourses_CourseNotFoundGiven_ReturnsCourseNotFoundResponse() {
		// ...
		$expected = [
			'status_code' => 404,
			'error_code' => 'sensei_course_student_batch_action_missing_course',
		];
		self::assertSame( $expected, $this->exportErrorResponse( $response ) );
	}

	private function exportErrorResponse( $response ): array {
		return [
			'status_code' => $response->get_status(),
			'error_code' => $response->get_data()['code'] ?? null,
		];
	}
```